### ISSUE TITLE
Change sensor priority so that the dvl depth always picks up exactly where the depth sensor left off

### ISSUE THREAD

**Issue by MrMondrian** [mcgill-robotics/AUV-2025#266]
Currently if the depth sensor goes offline, then when the dvl depth takes over it will jump to whatever the dvl thinks the depth is. Instead an offset should be applied to prevent the jump from occuring

**Comment by antoinedang** [comment]
NOTE: we could also apply something similar to DVL/IMU (I think going from IMU -> DVL should be good already because it uses the IMU's reference frame to correct, but DVL->IMU will jump). Correct me if im wrong @MrMondrian

**Comment by MrMondrian** [comment]
You're right, but is this case it might be better to let it jump because the IMU's quaternion is what we trust more. I'm imagining things going really weirdly if they flip flop multiple times in a row. Letting it jump could be dangerous though. @antoinedang

**Comment by antoinedang** [comment]
> You're right, but is this case it might be better to let it jump because the IMU's quaternion is what we trust more. I'm imagining things going really weirdly if they flip flop multiple times in a row. Letting it jump could be dangerous though. @antoinedang

That's true, let's keep it to just depth then. If it becomes a problem that they jump we could implement some sort of damping/interpolation between the two (would probably be over-engineering it tho lol). For now I agree lets not fix that problem unless it becomes one and just do depth
